package.json

```diff
@@ -36,19 +36,21 @@
   },
   "homepage": "https://github.com/mulesoft-labs/osprey-mock-service",
   "devDependencies": {
-    "chai": "^3.0.0",
-    "istanbul": "^0.4.1",
-    "mocha": "^2.1.0",
+    "chai": "^3.5.0",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.1.0",
     "pre-commit": "^1.0.1",
-    "standard": "^6.0.4"
+    "standard": "^8.3.0",
+    "popsicle": "^8.2.0",
+    "popsicle-server": "^2.0.0"
   },
   "dependencies": {
     "finalhandler": "^0.4.0",
     "morgan": "^1.7.0",
     "negotiator": "^0.6.0",
-    "osprey": "^0.3.0",
+    "osprey": "mulesoft/osprey#raml1.0",
```

Comment:
Once the new version is released, remind to change this reference to the corresponding version (0.4.0 I guess from https://github.com/mulesoft/osprey/pull/116/files)